lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -228,7 +228,7 @@ module InternalConstants
     Fluent::Plugin.register_output('google_cloud', self)
 
     PLUGIN_NAME = 'Fluentd Google Cloud Logging plugin'.freeze
-    PLUGIN_VERSION = '0.6.23'.freeze
+    PLUGIN_VERSION = '0.7.0'.freeze
```

Comment:
See #245.

Reply:
Great. I will rebase when that's merged.